plugins/database/postgresql/postgresql.go

```diff
@@ -529,7 +529,7 @@ func (p *PostgreSQL) defaultDeleteUser(ctx context.Context, username string) err
 		}
 		revocationStmts = append(revocationStmts, fmt.Sprintf(
 			`REVOKE ALL PRIVILEGES ON ALL TABLES IN SCHEMA %s FROM %s;`,
-			(schema),
+			dbutil.QuoteIdentifier(schema),
```

Comment:
I am not so sure this is a straightforward change. Is this backwards compatible? It doesn't appear to be so based on http://www.postgresql.org/docs/9.2/static/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS

> Quoting an identifier also makes it case-sensitive, whereas unquoted names are always folded to lower case.

So we might break users that depend on case-insensitivity.

Reply:
hmm, it looks like we already quote the schema name in the next revoke call here: https://github.com/hashicorp/vault/blame/2db2a9fb5d890b213a2c05aa5698c63560399774/plugins/database/postgresql/postgresql.go#L537

Reply:
Yeah it used to be quoted but it was (unintentionally I guess) dropped in this PR: https://github.com/hashicorp/vault/pull/15343

Reply:
Thanks! Do you mind adding a changelog entry?

`changelog/28519.txt`

```release-note:bug
database/postgresql: Fix potential error revoking privileges in postgresql database secrets engine when a schema contains special characters
```

Reply:
Added